cinema/serializers.py

```diff
@@ -0,0 +1,24 @@
+from rest_framework import serializers
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
```

Comment:
Consider using `serializers.ModelSerializer` instead of `serializers.Serializer` for the `MovieSerializer` class. `ModelSerializer` automatically generates the `create` and `update` methods and handles model fields more efficiently.